src/test/java/io/pravega/connectors/flink/sink/PravegaSinkWriterITCase.java

```diff
@@ -146,6 +148,21 @@ private static class SinkInitContext implements Sink.InitContext {
         SinkInitContext(
                 SinkWriterMetricGroup metricGroup) {
             this.metricGroup = metricGroup;
+                }
```

Comment:
Indentation not correct here.

Reply:
Hi Brian, The indentation has been fixed.